pkg/capabilities/pb/capabilities.proto

```diff
@@ -66,3 +66,17 @@ service CallbackExecutable {
   rpc UnregisterFromWorkflow (UnregisterFromWorkflowRequest) returns (google.protobuf.Empty) {}
   rpc Execute(CapabilityRequest) returns (stream CapabilityResponse) {}
 }
+
+service StandardCapability {
+  rpc Initialise(InitialiseRequest) returns (google.protobuf.Empty) {}
+}
+
+message InitialiseRequest {
+  string config = 1;
+  uint32 errorLogID = 2;
```

Comment:
the proto convention is snake case. iirc, it would effect the generated code https://protobuf.dev/programming-guides/style/#message-field-names